eslint.config.js

```diff
@@ -0,0 +1,50 @@
+import { fileURLToPath } from 'node:url';
+import { FlatCompat } from '@eslint/eslintrc';
+import { includeIgnoreFile } from '@eslint/compat';
+import js from '@eslint/js';
+import path from 'node:path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+const compat = new FlatCompat({
+	baseDirectory: __dirname,
+	recommendedConfig: js.configs.recommended,
+	allConfig: js.configs.all
+});
+const gitignorePath = path.resolve(__dirname, '.gitignore');
+const files = ['**/*.js', '**/*.html'];
+export default [
+	includeIgnoreFile(gitignorePath),
+	...compat.extends('brightspace/lit-config').map(c => ({
+		...c,
+		files
+	})),
+	{
+		languageOptions: {
+			globals: {
+				Prism: false,
+			},
+		},
+	},
+	{
+		files,
+		rules: {
+			'no-restricted-syntax': [0, 'CatchClause[param=null]']
+		},
+	},
+	{
+		files: ['**/demo/**.html'],
+		rules: {
+			'no-console': 0,
+		},
+	},
+	...compat.extends('brightspace/testing-config').map(config => ({
+		...config,
+		files: ['**/test/*'],
+	})),
+	...compat.extends('brightspace/node-config').map(config => ({
+		...config,
+		files: ['cli/*'],
+		rules: { 'no-console':0 }
```

Comment:
Can we just double-check that the node config doesn't already allow console logging?

Reply:
[It doesn't](https://github.com/Brightspace/eslint-config-brightspace/blob/main/node-config.js), maybe it should. this could be included with the changes to demo linting too.

Reply:
Cool agreed... logging in Node seems like an OK thing to want to do.